core/gui/src/app/workspace/component/left-panel/versions-list/versions-list.component.ts

```diff
@@ -59,7 +64,11 @@ export class VersionsListComponent implements OnInit {
       });
   }
 
-  getVersion(vid: number) {
+  getVersion(vid: number, displayedVersionId: number, index: number) {
+    this.workflowVersionService.setSelectedVersionId(vid);
```

Comment:
I think it is better to set the version id and displayed version id inside the API `WorkflowVersionService.displayParticularVersion`. Please remove the setters.